graphql-dgs/src/main/kotlin/com/netflix/graphql/dgs/internal/BaseDgsQueryExecutor.kt

```diff
@@ -118,7 +119,12 @@ object BaseDgsQueryExecutor {
                 .extensions(extensions.orEmpty())
                 .build()
             graphQLContextFuture.complete(executionInput.graphQLContext)
-            graphQL.executeAsync(executionInput)
+            graphQL.executeAsync(executionInput).thenApply {
```

Comment:
`whenComplete` might be safer, although I don't know under what circumstances `executeAsync` ever really fails; maybe cancellation?